src/utils.tsx

```diff
@@ -94,3 +95,22 @@ export function uuidV4 ():string {
     return value.toString(16)
   })
 }
+
+export function useNonInitialEffect (effect: EffectCallback, deps?: DependencyList): void {
+  const initialRender = useRef(true)
+
+  useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    let effectReturns: void | (() => void) = () => {}
+
+    if (initialRender.current) {
```

Comment:
I would have prefer an early return statement, i think it would have simplified the code. Something like:
```js
if (initialRender.current) {
  initialRender.current = false
  return
}
if (typeof effect === 'function') {
  return effect();
}
```
no need for the effectReturns variable